sct.py

```diff
@@ -1390,6 +1391,9 @@ def create_test_release_jobs(branch, username, password, sct_branch, sct_repo):
         if group_name == 'disabled_raft':
             for jenkins_file in glob.glob(f'{base_path}/consistent_cluster_management_disabled/*'):
                 server.create_pipeline_job(jenkins_file, group_name)
+        if group_name == 'upgrade-with-raft':
+            for jenkins_file in glob.glob(f'{base_path}/{group_name}/*.jenkinsfile'):
```

Comment:
Looks like it fails.
Need to precreate `{base_path}/{group_name}` dirs first and only then jenkins files.

Reply:
it was intentional to have it failing in the link above... it was to see when the code was calling for the files i want